main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+)
+
+var (
+	airtableApiKey = os.Getenv("AIRTABLE_API_KEY")
+	airtableBaseId = os.Getenv("AIRTABLE_BASE_ID")
+)
+
+func healthHandler(w http.ResponseWriter, _ *http.Request) {
+	w.Write([]byte("PONG"))
+}
+
+func main() {
+	if airtableApiKey == "" {
+		panic("AIRTABLE_API_KEY must be specified")
+	}
+	if airtableBaseId == "" {
+		panic("AIRTABLE_BASE_ID must be specified")
+	}
+
+	client, err := NewCachingRedirectClient(airtableApiKey, airtableBaseId)
+	if err != nil {
+		panic(err)
```

Comment:
when this is returning an error, you can augment the error with more information:

    fmt.Errorf("failed to build redirect client: %v", err)